src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_after_oog_after_deeperFiller.json

```diff
@@ -0,0 +1,98 @@
+{
+    "returndatasize_after_oog_after_deeper" : {
+        "_info" : {
+            "commment" : "transaction calls A (CALL B(CALL C(RETURN) OOG) 'check buffers')"
+        },
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance": "0x1000000000",
+                "code": "",
+                "nonce": "0x0",
+                "storage": {}
+            },
+            "0x1000000000000000000000000000000000000001": {
+                "balance" : "0x00",
+                "code": "{ (seq (SSTORE 2 (CALL 100000 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 32)) (SSTORE 0 (RETURNDATASIZE))) (SSTORE 1 (MLOAD 0))}",
+                "nonce" : "0x00",
+                "storage" : {
+                    "0x00" : "0xffffffff",
+                    "0x01" : "0xffffffff",
+                    "0x02" : "0xffffffff"
+                }
+            },
+            "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x100000000000",
+                "code" : "{ (seq (CALL 100000 0xbb00000000000000000000000000000000000000 0 0 0 0 0) (while 1 (SSTORE 0 1)) )}",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0xbb00000000000000000000000000000000000000" : {
+                "balance" : "0x00",
+                "code" : "{ (seq (STOP) )}",
```

Comment:
this should return some non empty data. via revert or return

Reply:
That makes sense.  Will do.

Reply:
Done.